services/email/templates/customer/plain/delivery.mus

```diff
@@ -5,16 +5,12 @@ We are currently experiencing very high volumes of requests
 
```

Comment:
in the ticket it says they want this

`You have reported that your BRP hasn't arrived`

Is this already there in the heading?

Not a must but nice to have a screenshot of the change

Reply:
Yeah, that is set in the heading. The email isn't coming through to my digital one.